app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
```

Comment:
consider a case, where I as an owner purchased a membership. 
After a month, I decide to add one of my friends. After another month, I add two more. How will I know from the db that when my friends were added to the membership by me

Reply:
Sir, now SubscriptionUsers model is created and we can track adding time thorugh its created_at